tests/components/google/conftest.py

```diff
@@ -29,8 +29,7 @@
 
 ApiResult = Callable[[dict[str, Any]], None]
 ComponentSetup = Callable[[], Awaitable[bool]]
-_T = TypeVar("_T")
-YieldFixture = Generator[_T, None, None]
+type AsyncYieldFixture[_T] = AsyncGenerator[_T, None]
```

Comment:
The alias is only used to annotate async fixture functions. Thus it should be `AsyncGenerator` instead of `Generator`.